metadata/src/main/java/org/apache/kafka/metalog/MetaLogManager.java

```diff
@@ -50,13 +50,30 @@
      * offset before renouncing its leadership.  The listener should determine this by
      * monitoring the committed offsets.
      *
-     * @param epoch         The controller epoch.
-     * @param batch         The batch of messages to write.
+     * @param epoch         the controller epoch
+     * @param batch         the batch of messages to write
```

Comment:
The other methods in the class consistently end `@param` with a period.

Reply:
These are phrases hence no period. For example none of the javadoc for the Java library have periods.

I only modified the javadoc that I needed to update to not distract from the PR. We should do another pass and fix all of the comments.